lib/Player.tsx

```diff
@@ -339,6 +355,7 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
                 parameters={parameters}
                 videoProperties={videoProperties}
                 refresh={refresh}
+                volume={volume}
```

Comment:
Can be removed (in case if it's possible to use `volume` from `videoProperties` instead)

Reply:
I found out that it is not. videoProperties doesn't update on change, only on play/pause.

Reply:
I did however make sure the is an audioTrack present.